### DISCUSSION TITLE
x-www-form-urlencoded body conversion

### DISCUSSION THREAD

**Discussion by shubhambajad** [apideck-libraries/portman#324] [Q&A] [answered]
Hi Guys,

Currently in our postman collection we are using x-www-form-urlencoded body instead of json body.

So, we need to add a value for all the parameters for x-www-form-urlencoded body.

so do portman provide any kind of solution for that? As I have used overwriteRequestBody but is is not working I think it is only for json body. So can you please provide a help for the same.

**Comment by thim81** [comment]
hi @shubhambajad 

Currently Portman, only supports JSON body overwrites.

We have plans to bring x-www-form support in the future, but there is not concrete timeline for this.
If you want to contribute, feel free to create a PR and we will gladly help out and assist with the implementation of this feature.

I've created an "issue" of your request to register this feature request.

**Comment by thim81** [comment]
hi @shubhambajad 

The next version of Portman will include support for overwrites of x-www-form & url-encoded forms (see #413 for more context).

**Comment by thim81** [comment, marked as answer]
In Portman 1.19, we introduced the option use overwriteRequestBody for the types "form-data" & "x-www-form-urlencoded"
